ckanext/justicehub_theme/templates/organization/read.html

```diff
@@ -0,0 +1,40 @@
+{% ckan_extends %}
+
+{% block primary_content_inner %}
+    {% block groups_search_form %}
+      {% set facets = {
+        'fields': c.fields_grouped,
+        'search': c.search_facets,
+        'titles': c.facet_titles,
+        'translated_fields': c.translated_fields,
+        'remove_field': c.remove_field }
+      %}
+      {% set sorting = [
+        (_('Relevance'), 'score desc, metadata_modified desc'),
+        (_('Name Ascending'), 'title_string asc'),
+        (_('Name Descending'), 'title_string desc'),
+        (_('Last Modified'), 'metadata_modified desc'),
+        (_('Popular'), 'views_recent desc') if g.tracking_enabled else (false, false) ]
```

Comment:
can we instead have `Name (A-Z)` and `Name(Z-A)` @Desingneer had suggested to have these in sorting from a usability perspective.